main.c

```diff
@@ -202,7 +186,7 @@ void forwardReactiveFlag(void) {
 /*
  * Execute action based on a message
  */
-void executeMsg(msg_t msg) {
+inline void executeMsg(msg_t msg) {
```

Comment:
so inline actually doesn't guarantee it gets inlined.

Reply:
added static.